includes/helpers/class-accredible-learndash-admin-form-helper.php

```diff
@@ -32,11 +32,11 @@ public static function value_attr( $obj, $key ) {
 		 * Returns value if present else shows default value.
 		 *
 		 * @param mixed $value value.
-		 * @param mixed $default default value.
+		 * @param mixed $default_value default value.
 		 */
-		public static function html( $value, $default = 'N/A' ) {
+		public static function html( $value, $default_value = 'N/A' ) {
 			if ( empty( $value ) ) {
-				$value = $default;
+				$value = $default_value;
```

Comment:
To fix the following PHPCS warning:

<img width="855" alt="Screenshot 2024-10-03 at 11 03 35" src="https://github.com/user-attachments/assets/3f9e0133-d19a-4804-b39d-ee57e1769468">